util/find-rules-without-test/README.md

```diff
@@ -0,0 +1,31 @@
+# Find the rules without test cases
+
+This page describes how can you find the rules without any test cases
+
+## Goal
+
+The main goal that we must to have at least regression test for all relevant REQUEST-* rules. (In this context, the PL controll rules are not relevant, because they do not need to have tests.)
```

Comment:
The main goal **is** that we **must have** (without the to) at least **one** regression test for all relevant REQUEST- * rules. (In this context, the PL **control** rules are not relevant, because they do not need to have tests.)

Reply:
Thanks, fixed.